index.bs

```diff
@@ -2231,6 +2237,139 @@ and makes it clear when the state described by the flags is reset.
+we fail to find a way
+to interoperably specify a feature
+that every implementor agrees
+is implementable in their engine.
+This is a failure of the standards process.
+
+There are many possibile next steps when this happens.
+Choosing a way forward should be guided by
+[what is best for the end user](#priority-of-constituencies).
+
+First,
+examine the nature of the implementability concerns.
+Perhaps the implementor identified potential end-user harm
+and decided that the feature should not be implemented.
+In this situation,
+it may be best to not specify the feature,
+and for any implementors who have shipped the feature
+to un-ship it.
+Un-shipping features may result in user and author
+confusion.
+
+<div class=example id=sharedarraybuffer>
+
+When the Spectre attack was discovered,
+{{SharedArrayBuffer}} was unshipped in every implementation
+because it can be used to create a high-precision timer
+that could be used for a Spectre attack.
+
+</div>
+
+Sometimes the same API is implementable in every engine,
+but its behavior cannot be made to be fully interoperable.
+Whenever behavior differs between implementations,
+the end-user impact must be considered.
+Depending on the severity and frequency of the behavior differences,
+this may or may not be an acceptable outcome.
+One downside to this approach
+is that the difference in behavior is not [feature detectable](#feature-detect).
+On the other hand,
+if something changes in the future
+that enables all implementations to converge on the same behavior,
+sites will not need to be updated
+to take advantage of this.
+
+Note: Authors may assume the behavior of the dominant implementation is correct
+and any other behavior is buggy,
+which may further entrench the dominant implementaiton.
+
+<div class=example id=unknown1>
+
+The 'backdrop-filter' property is known to have a number of visible
+behavior differences between implementations.
+At the present time,
+those working on it believe
+the benefits of sharing an API
+outweigh the interoperability costs of the differences.
+
+</div>
+
+If the behavior differences are serious enough,
+it may be preferable to specify different APIs.
+
+In this scenario,
+each implementor goes with the alternative
+that they are able to implement.
+If implementations are eventually able to converge on the same behavior,
+both APIs may need to be supported indefinitely,
+which can increase implementation maintenance costs.
+
+Exposing two different APIs
+that are intended to address the same use cases
+can be confusing to authors.
+Authors can use [feature detection](#feature-detect)
+to write different code for different browsers.
+
+Note: There is a risk that authors may assume
+the API variant supported by the dominant implementation
+is the "correct" one,
+or they may simply be unaware of the API variants supported
+in implementations they do not use regularly,
+which may further entrench the dominant implementaiton.
```

Comment:
```suggestion
Note: This may have consequences for the browser ecosystem.
Authors may choose to only support 
the version implemented in the most widely-used browser engine,
meaning users of browsers with different engines
have a poor experience.
This in turn may cause users to feel they have no choice
but to use the more widely-used browser.
```